src/tests/electron/main.test.ts

```diff
@@ -18,59 +18,49 @@ describe('Electron E2E', () => {
         return app.start();
     });
 
+    // spectron wraps calls to electron APIs as promises. Unfortunately, only electron typings are used,
+    // so tslint thinks some of the methods do not return promises.
+    // tslint:disable: await-promise
+
     // tslint:disable-next-line:typedef
     async function ensureAppIsInDeviceConnectionDialog() {
-        // spectron wraps calls to electron APIs as promises. Unfortunately, only electron typings are used,
-        // so tslint thinks some of the methods do not return promises.
-        // tslint:disable: await-promise
-        expect(await app.browserWindow.isVisible()).toBe(true);
-        expect(await app.client.getWindowCount()).toBe(1);
+        const webDriverClient: WebdriverIO.Client<void> = app.client;
+        await webDriverClient.waitForVisible(CommonSelectors.deviceViewContainer, 5000);
```

Comment:
thought: Is this timeout different than the default timeout for your end-to-end tests? We can definitely store this in a common place and maybe have each test a common timeout period, like we do in our web e2e.

Reply:
Fixed